editor/src/clj/editor/model.clj

```diff
@@ -74,7 +84,11 @@
                     pb
                     (map (fn [[label res]]
                            [label (resource/proj-path (get dep-resources res))])
-                         (:dep-resources user-data)))]
+                         (:dep-resources user-data)))
+        textures (map make-texture (:textures pb))
+        material (make-material "default" (:material pb) textures)
+        pb (assoc pb :materials (list material))
+        pb (dissoc pb :material :textures)]
```

Comment:
```suggestion
        textures (mapv make-texture (:textures pb))
        material (make-material "default" (:material pb) textures)
        pb (assoc pb :materials [material])
        pb (dissoc pb :material :textures)]
```
* Prefer `mapv` when collecting the textures since we want a realized vector, not a lazyily produced sequence.
* Use a single-element vector instead of a `list` for the `:materials`.